### ISSUE TITLE
No specification for the interaction between ANGLE_instanced_arrays and OES_vertex_array_object

### ISSUE THREAD

**Issue by greggman** [KhronosGroup/WebGL#1955]
This is probably not a big deal but .... AFAICT there's no specification for the interaction between `ANGLE_instanced_arrays` and `OES_vertex_array_object` (and probably no tests since both would have to be enabled)

In other words `ANGLE_instance_arrays` adds a divisor setting per attribute. `OES_vertex_array_object` moves the attributes to separate objects. But `OES_vertex_arrays_object` does not mention the divisor. At best it says 

>  All state related to the definition of the data used by the vertex  processor is encapsulated in a vertex array object.

But it goes on to mention 

> The resulting vertex array object is a new state vector, comprising all the state values (listed in Table 6.2, except ARRAY_BUFFER_BINDING):
>
>          VERTEX_ATTRIB_ARRAY_ENABLED,
>          VERTEX_ATTRIB_ARRAY_SIZE,
>          VERTEX_ATTRIB_ARRAY_STRIDE,
>          VERTEX_ATTRIB_ARRAY_TYPE,
>          VERTEX_ATTRIB_ARRAY_NORMALIZED,
>          VERTEX_ATTRIB_ARRAY_POINTER,
>          ELEMENT_ARRAY_BUFFER_BINDING,
 >          VERTEX_ATTRIB_ARRAY_BUFFER_BINDING.

No mention of `VERTEX_ATTRIB_ARRAY_DIVISOR`.

In other words it seems ambiguous whether the divisors become part of the vertex array object's state or if it's global state.

ES 3.0 makes it clear it's part of the vertex array object's state.

I don't know if this needs to be cleared up. I suppose it belongs in the OpenGL mailing list?

**Comment by kenrussell** [comment]
Sorry for the confusion. I'm pretty sure this was discovered as a problem while implementing WebGL 2.0, and yes, the vertex attribute divisor is part of the vertex array object's state. Yes, these extensions should refer to each other. CC'ing @zhenyao @vonture @null77 for any corrections.
